lib/styles.less

```diff
@@ -313,7 +315,10 @@ body {
         font-weight: @typography_notes_fontWeight;
         text-transform: @typography_notes_textTransform;
         font-size: @typography_notes_fontSize;
-        line-height: @typography_notes_lineHeight;
+        font-size: @typography_aboveFooter_fontSize;
```

Comment:
I haven't looked at how this gets used, but why is this used here when it's also already being used in the above `.dw-above-footer` block?

Reply:
Ah, great very well spotted! looks like copy paste mistake to me, just fixed it thank you eagle-eye...van

Reply:
You're welcome 😄